src/ui/StackedBarChart.js

```diff
@@ -103,8 +111,8 @@ WGL.ui.StackedBarChart = function(m, div_id, x_label, filterId, params) {
     //set arrow height for first
     this.setArrowHeight(arrowTan);
 
-    var cols = [ "#ff8c00", "#7b6888", "#98abc5" ];
-    //var cols = [ "#ff8c00", "#3182bd", "#98abc5" ];
+    var cols = [ "#fd9a2f", "#50a7f9", "#dce2e0" ];
+
```

Comment:
This will change the default colour of the histogram? Can you leave the defaults and enable to set this from the example app (init.js)?

Reply:
Done, back to default colors